ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/upgrade/eos-intended-config.j2

```diff
@@ -0,0 +1,252 @@
+{# Device Configuration #}
```

Comment:
Is creation of this file expected? I don't quite see what it has to do with the feature in the PR.

Reply:
I'm not sure about this file. Can you help pls, @tgodaA

Reply:
Please delete the file from your branch and commit.